src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -177,7 +177,7 @@ namespace cryptonote
   , "Run a program for each new block, '%s' will be replaced by the block hash"
   , ""
   };
-  static const command_line::arg_descriptor<bool> arg_prune_blockchain = {
+  [[maybe_unused]] static const command_line::arg_descriptor<bool> arg_prune_blockchain = {
```

Comment:
it might be better to permanently delete that command_line option due to SN data pruning will destroy database